client/src/app/components/Autocomplete.tsx

```diff
@@ -0,0 +1,288 @@
+import React from "react";
+import {
+  Label,
+  Flex,
+  FlexItem,
+  Menu,
+  MenuContent,
+  MenuItem,
+  MenuList,
+  Popper,
+  SearchInput,
+  Divider,
+} from "@patternfly/react-core";
+
+export interface IAutocompleteProps {
+  options?: string[];
+  value?: string;
+  labelColor?:
+    | "blue"
+    | "cyan"
+    | "green"
+    | "orange"
+    | "purple"
+    | "red"
+    | "grey"
+    | "gold"
+    | undefined;
+  selections?: string[];
+}
+
+export const Autocomplete: React.FC<IAutocompleteProps> = ({
+  // TODO: data just for testing purposes, should be removed
+  options = ["Cluster", "Kind", "Label", "Name", "Namespace", "Status"],
+  value,
+  labelColor,
+  selections = [],
+}) => {
+  const [inputValue, setInputValue] = React.useState(value || "");
```

Comment:
You can also assign the value prop with a default value of "" and remove the `|| ""` at the end